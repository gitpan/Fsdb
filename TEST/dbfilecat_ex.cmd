prog='dbfilecat'
args='--input - --input TEST/dbmerge_ex_b.in'
in=TEST/dbmerge_ex.in
cmp='diff -cb '
