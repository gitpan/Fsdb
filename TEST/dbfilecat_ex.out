#fsdb cid cname
11	numanal
10	pascal
12	os
13	statistics
#   | dbfilecat --input - --input TEST/dbmerge_ex_b.in
